core/backend/src/Relationship.ts

```diff
@@ -197,13 +197,21 @@ export class Relationships {
    */
   public createInstance(props: RelationshipProps): Relationship { return this._iModel.constructEntity<Relationship>(props); }
 
-  /** Insert a new relationship instance into the iModel.
+  /** Check classFullName to ensure it is a link table relationship class. */
+  private checkRelationshipClass(classFullName: string) {
+    if (!this._iModel.nativeDb.isLinkTableRelationship(classFullName.replace(".", ":"))) {
+      throw new IModelError(DbResult.BE_SQLITE_ERROR, `Class "${classFullName} must be a relationship class and it should be subclass of BisCore:ElementRefersToElements or BisCore:ElementDrivesElement."`, Logger.logWarning, loggerCategory);
```

Comment:
Seems like the `"` characters are in the wrong place. Perhaps just around `${classFullName}`?

Reply:
[fixed ](https://github.com/imodeljs/imodeljs/pull/324/commits/6effc8c5bf7c9261712dce03b193add82461f0a4)